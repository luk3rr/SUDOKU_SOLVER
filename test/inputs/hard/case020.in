000004508 049015003 530008014 200800000 000000801 004509300 000007109 460000000 008000007 
