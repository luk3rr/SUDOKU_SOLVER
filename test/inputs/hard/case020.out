Solving the following grid:
0 0 0 | 0 0 4 | 5 0 8 
0 4 9 | 0 1 5 | 0 0 3 
5 3 0 | 0 0 8 | 0 1 4 
------+-------+------
2 0 0 | 8 0 0 | 0 0 0 
0 0 0 | 0 0 0 | 8 0 1 
0 0 4 | 5 0 9 | 3 0 0 
------+-------+------
0 0 0 | 0 0 7 | 1 0 9 
4 6 0 | 0 0 0 | 0 0 0 
0 0 8 | 0 0 0 | 0 0 7 


Solution found :')

7 2 1 | 6 3 4 | 5 9 8 
8 4 9 | 7 1 5 | 6 2 3 
5 3 6 | 9 2 8 | 7 1 4 
------+-------+------
2 7 3 | 8 4 1 | 9 5 6 
6 9 5 | 3 7 2 | 8 4 1 
1 8 4 | 5 6 9 | 3 7 2 
------+-------+------
3 5 2 | 4 8 7 | 1 6 9 
4 6 7 | 1 9 3 | 2 8 5 
9 1 8 | 2 5 6 | 4 3 7 

Algorithm: BFS
Total time: 1015 ms
Total expanded states: 154260
